tests/providers/amazon/aws/hooks/test_emr.py

```diff
@@ -181,6 +182,11 @@ def test_get_cluster_id_by_name(self):
             {"Name": "test_cluster", "Instances": {"KeepJobFlowAliveWhenNoSteps": True}}
         )
 
+        for i in range(51):
+            hook.create_job_flow(
+                {"Name": "test_cluster" + str(i % 50), "Instances": {"KeepJobFlowAliveWhenNoSteps": True}}
```

Comment:
Did you use `i % 50` so that the 0 case would be in the list twice? If so it's a bit sneaky and non-obvious IMHO.

Reply:
Yes I did. Was just a fast way to test, but I can add a comment to make it clearer or just hardcode a loop around a different string.

Reply:
Addressed